spec/spec_helper.rb

```diff
@@ -6,6 +8,7 @@
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 
 require 'webmock/rspec'
+require 'zendesk_apps_tools'
```

Comment:
the actual app does this to setup autoload paths, so we should do it in tests. Otherwise I'm ending up in a circular require trap.